### DISCUSSION TITLE
Model-viewer-effects Add-on with Vanilla JS ?

### DISCUSSION THREAD

**Discussion by pyjacques** [google/model-viewer#4280] [Q&A]
I have a working 3D experienced using pure javascript with this UMD version.

But i would like to use the the effect addon ^^
<script src="https://cdn.jsdelivr.net/npm/@google/model-viewer@3.1.1/dist/model-viewer-umd.min.js"></script>
But the official exemple (https://modelviewer.dev/examples/postprocessing/) isn't working for me in vanilla JS : 
```
<script async src="https://ga.jspm.io/npm:es-module-shims@1.7.1/dist/es-module-shims.js"></script>
    <script type="importmap">
      {
        "imports": {
          "three": "https://cdn.jsdelivr.net/npm/three@^0.152.1/build/three.module.min.js"
        }
      }
    </script>
    <script type="module" src="https://cdn.jsdelivr.net/npm/@google/model-viewer@3.1.1/dist/model-viewer-umd.min.js"></script>
    <script type="module" src="https://cdn.jsdelivr.net/npm/@beilinson/model-viewer-effects@1.0.0/dist/model-viewer-effects.min.js"></script>
```
![image](https://github.com/google/model-viewer/assets/44160710/77ceb2bd-fc72-4479-9ade-ecd620e30657)
Any tips ? Thanks =)

**Comment by elalish** [comment]
That's because post-processing is still pre-release; you can test following these instructions: https://modelviewer.dev/docs/faq.html#entrydocs-general-questions-api
